README.md

```diff
@@ -27,4 +27,76 @@ module.exports = {
 };
 ```
 
-For more, see [the Prettier documentation](https://prettier.io/docs/en/configuration.html#sharing-configurations).
+For more information, see [the Prettier documentation](https://prettier.io/docs/en/configuration.html#sharing-configurations).
+
+### With ESLint
+
+`npm install --save-dev eslint-config-prettier`
+
+In your ESLint configuration, add `"prettier"` to `extends`. For example, with `.eslintrc.json` and [the 6 River ESLint configuration](https://github.com/6RiverSystems/eslint-config-6river):
+
+```json
+{
+	"extends": ["6river", "prettier"]
+}
+```
+
+If you are using Typescript and the ESLint plugin `@typescript-eslint/eslint-plugin`, add `"prettier/@typescript-eslint"`
+to `extends` as well. For example, with `.eslintrc.json` and [the 6 River ESLint configuration](https://github.com/6RiverSystems/eslint-config-6river):
+
+```json
+{
+	"extends": ["6river/typescript", "prettier", "prettier/@typescript-eslint"]
```

Comment:
When I was cargo culting from one of the MFP projects into my playground project, I ended up adding `plugin:import/typescript` here too.  Was that unnecessary?  It was not immediately clear what that did at the time.

Reply:
@mgabeler-lee-6rs That's actually separate, but still cool. The plugin docs are [here](https://github.com/benmosher/eslint-plugin-import). I was piloting it in MFP UI because it bugged me that we didn't have any rules around import ordering.